lib/views/BinnedMeanGraph.js

```diff
@@ -93,7 +93,7 @@ class BinnedMeanGraph extends Graph {
         var xMax = width - margins.right;
         var xLabelFormat = chartOpts.xLabelFormat;
 
-        function updateGuides(circleData, circleIdx) {
+        const updateGuides = (circleData, circleIdx) => {
```

Comment:
Babel doesn't allow `super` calls outside of classes/unbound functions anymore (same comment applies to the next three files)